### DISCUSSION TITLE
Webhook Listener

### DISCUSSION THREAD

**Discussion by aurabhi** [ToolJet/ToolJet#2955] [Q&A]
Hi, Is there a way to receive http notifications from an external app and then on receiving a notification save it to a data store, say Postgre?

**Comment by withshubh** [comment]
Hey @aurabhi 👋 
This feature is currently not available. I'll reach out to you if we ship it in near future.

**Reply by justinr1234** [reply]
@withshubh are you saying it isn’t possible to receive incoming http requests at all?

**Reply by withshubh** [reply]
Hey @justinr1234 - As of now it's not possible but it's in our roadmap. We'll update here once we have an ETA.

**Comment by RammusXu** [comment]
I hope this feature can be implemented. It is impossible to replace Airtable with ToolJet without the Incoming Webhook feature.

**Comment by russell-hutson-jp** [comment]
It would be great if we could build APIs in ToolJet
